pkg/config/parse_test.go

```diff
@@ -7,21 +7,65 @@ import (
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 )
 
 const exampleConfigPath = "../../config.example.toml"
 
+func compareConfigs(parsedConf *Config, expConf *Config, t *testing.T) {
+	opts := cmpopts.IgnoreTypes(StorageConfig{}, ProxyConfig{}, OlympusConfig{})
+	eq := cmp.Equal(parsedConf, expConf, opts)
+	if !eq {
+		t.Errorf("Parsed Example configuration did not match expected values. Expected: %+v. Actual: %+v", expConf, parsedConf)
+	}
+	eq = cmp.Equal(parsedConf.Proxy, expConf.Proxy)
+	if !eq {
+		t.Errorf("Parsed Example Proxy configuration did not match expected values. Expected: %+v. Actual: %+v", expConf.Proxy, parsedConf.Proxy)
+	}
+	eq = cmp.Equal(parsedConf.Olympus, expConf.Olympus)
+	if !eq {
+		t.Errorf("Parsed Example Olympus configuration did not match expected values. Expected: %+v. Actual: %+v", expConf.Olympus, parsedConf.Olympus)
+	}
+	compareStorageConfigs(parsedConf.Storage, expConf.Storage, t)
+}
+
+func compareStorageConfigs(parsedStorage *StorageConfig, expStorage *StorageConfig, t *testing.T) {
+	eq := cmp.Equal(parsedStorage.CDN, expStorage.CDN)
+	if !eq {
+		t.Errorf("Parsed Example Storage configuration did not match expected values. Expected: %+v. Actual: %+v", expStorage.CDN, parsedStorage.CDN)
+	}
+	eq = cmp.Equal(parsedStorage.Mongo, expStorage.Mongo)
+	if !eq {
+		t.Errorf("Parsed Example Storage configuration did not match expected values. Expected: %+v. Actual: %+v", expStorage.Mongo, parsedStorage.Mongo)
+	}
+	eq = cmp.Equal(parsedStorage.Minio, expStorage.Minio)
+	if !eq {
+		t.Errorf("Parsed Example Storage configuration did not match expected values. Expected: %+v. Actual: %+v", expStorage.Minio, parsedStorage.Minio)
+	}
+	eq = cmp.Equal(parsedStorage.Disk, expStorage.Disk)
+	if !eq {
+		t.Errorf("Parsed Example Storage configuration did not match expected values. Expected: %+v. Actual: %+v", expStorage.Disk, parsedStorage.Disk)
+	}
+	eq = cmp.Equal(parsedStorage.GCP, expStorage.GCP)
+	if !eq {
+		t.Errorf("Parsed Example Storage configuration did not match expected values. Expected: %+v. Actual: %+v", expStorage.GCP, parsedStorage.GCP)
+	}
+}
+
 func TestEnvOverrides(t *testing.T) {
 
-	filterOff := false
 	expProxy := ProxyConfig{
 		StorageType:           "minio",
 		OlympusGlobalEndpoint: "mytikas.gomods.io",
 		RedisQueueAddress:     ":6380",
 		Port:                  ":7000",
-		FilterOff:             &filterOff,
+		FilterOff:             false,
 		BasicAuthUser:         "testuser",
 		BasicAuthPass:         "testpass",
+		ForceSSL:              true,
+		ValidatorHook:         "testhook.io",
+		PathPrefix:            "prefix",
+		NETRCPath:             "/test/path",
```

Comment:
path.Join so it work on Windows

Reply:
+1 on this. @rchakra3 it'll be important so that windows users can run tests. would you mind changing this?

Reply:
I've switched it everywhere else - this is actually a test to validate environment overrides (so it's a string match, not an actual path in the filesystem)